.circleci/config.yml

```diff
@@ -0,0 +1,80 @@
+version: 2.1
+orbs:
+  node: circleci/node@5.0.2
+jobs:
+  build:
+    executor: node/default
+    steps:
+      - checkout
+      - node/install-packages:
+          pkg-manager: npm
+      - run:
+          command: npm install
+          name: Install npm
+      - run:
+          command: sudo npm install -g ts-node
+          name: Install ts-node
+      - run:
+          command: npm run test
+          name: Run tests
+      - run:
+          command: ts-node -v
+          name: Check ts-node version to make sure it installed
+
+  lint:
+    executor: node/default
+    steps:
+      - checkout
+      - node/install-packages:
+          pkg-manager: npm
+      - run:
+          command: sudo npm install -g lint
```

Comment:
name needs update and also again this shouldn't be necessary? `npm run` scripts should automatically update PATH variables to use npm installed packages